modules/installation/pages/upgrade.adoc

```diff
@@ -85,6 +85,128 @@ $ gadmin restart all -y
 
 . Enable CRR on the DR cluster.
 
+[#_rolling_upgrade]
+== Rolling Upgrade (Preview)
+
+A rolling upgrade allows you to upgrade your TigerGraph installation with minimal downtime by upgrading nodes in batches, ensuring that at least one full replica of the cluster is online throughout the process. 
+
+[NOTE]
+====
+Rolling Upgrade is only available for HA clusters (High Availability) and is supported for maintenance releases (e.g., 4.2.0 to 4.2.1, 4.3.1 to 4.3.3). 
+Upgrading from 4.2.0 to 4.3.0 or any other major version is not supported.
+====
+
+During the rolling upgrade, certain operations are not supported, including installing queries, running schema changes, dropping all data, using `gsql --reset`, creating graphs, backup/restore, clearing graph stores, exporting/importing graphs, and installing functions. Additionally, occasional drops in queries per second (QPS) may occur. 
+The following services will not be upgraded during the rolling upgrade process: Zookeeper (ZK), Kafka, and ETCD.
+
+=== Rolling Upgrade Process
+
+. Download and Extract Latest Version
+
+* Download the latest maintenance release of TigerGraph.
+* Extract the tar file on your system.
+
+. Run Installation Script with Upgrade Flag
+
+Run the following installation script with the upgrade flag (`+-Ur+`) that was extracted from the tar file:
+
+[source, gsql]
+----
+$ ./install.sh -Ur
+----
+
+. Switching to New Version
+
+After the binaries and configuration files are installed, user will be prompted with the following message:
+
+[console]
+----
+"Do you want to switch platform to the new version now (it can be delayed to a later time): y/N"
+----
+
+. n/N option
+
+Selecting n/N will guide the user through the manual steps to switch to version 4.2.1, assuming the user is upgrading from 4.2.0.
+
+[console]
+----
+To switch platform to new version, please run 'gadmin upgrade 4.2.1'
+----
+
+. y/Y Option
+
+If users selects `y/Y' the installer will automatically switch TigerGraph to the latest maintenance version. 
+If any errors occur, the upgrade process will terminate immediately. Follow the instructions in the prompts to manually continue the upgrade or revert to the original version.
+
+. Option `-n`
```

Comment:
I think this option is for the original command, e.g. ` ./install.sh -Ur -n`
Is that right? It's not clear from the document.

It is confusing to have it in this location, because it is seems be a 3rd choice from the y/N option.
Maybe use a heading instead of a numbered list.